viewer/packages/pointcloud-styling/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@reveal/pointcloud-styling",
+  "private": true,
+  "main": "index.ts",
+  "scripts": {
+    "start": "yarn ws:serve --env testFixture=Rendering.VisualTest.ts",
```

Comment:
This seems wrong, I assume this should start the rendering visual test?

Reply:
Woops, you're right :sweat_smile: Just forgot to remove this after copying the package json